src/app/modules/main/chat_section/controller.nim

```diff
@@ -466,6 +468,9 @@ proc setActiveItem*(self: Controller, itemId: string) =
   let isSectionActive = self.getIsCurrentSectionActive()
   if not isSectionActive:
     return
+
+  self.asyncCheckAllChannelsPermissions()
```

Comment:
This is overkill. We shouldn't need to check all channels each time we switch section.

Reply:
Okay, I can see permissions are check inside the `activeItemSet` [here](https://github.com/status-im/status-desktop/blob/fix-view-and-post-channel-permission/src/app/modules/main/chat_section/module.nim#L532-L533)
This means, something is masking the permission check. Reverting this.